.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@
 # See https://pre-commit.com/hooks.html for more hooks
 # This should be the _latest_ version of python supported by us
 default_language_version:
-  python: python3.9
+  python: python3.11
```

Comment:
Revert this to `python3.9`. We've standardized this version internally across core contributors and do not find any benefit in upgrading now.

Reply:
What’s new in Python 3.11…
> Python 3.11 is between 10-60% faster than Python 3.10. On average, we measured a 1.25x speedup on the standard benchmark suite.